schemas/nsip-document.schema.json

```diff
@@ -187,6 +183,14 @@
 			"type": "string",
 			"examples": ["Scoping Option Report"],
 			"description": "Filter field to provide additional filtering"
-		}
+		},
+    "horizonFolderId": {
+      "type": "string",
+      "description": "The folder where the document exists"
+    },
+    "transcriptId": {
+      "type": "string",
+      "description": "The GUID of the transcript document (if one was provided)."
+    }
```

Comment:
Formatting seems off here?

(note to self, we should add prettier pre-commit hook!)